backends/advanced/src/advanced_omi_backend/auth.py

```
@@ -224,6 +224,9 @@ async def create_admin_user_if_needed():
         existing_admin = await user_db.get_by_email(ADMIN_EMAIL)
 
         if existing_admin:
+            logger.error(f"🔍 DEBUG existing_admin.id = {existing_admin.id}, type = {type(existing_admin.id)}")
+            logger.error(f"🔍 DEBUG str(existing_admin.id) = {str(existing_admin.id)}")
```

Ruff (0.14.13)

Use explicit conversion flag

Replace with conversion flag

(RUF010)